launchpad/api.go

```diff
@@ -0,0 +1,98 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package launchpad
```

Comment:
I'd say it is too early to create abstractions. We follow AHA https://en.wikipedia.org/wiki/Don%27t_repeat_yourself, so only create abstractions if really required. A single HTTP post doesn't warrent an abstraction yet.

Reply:
Suggest just copy pasting `postHTTP` into both `cmd` and `dkg` packages

Reply:
Or rather, I do think `obolapi` should be a top level package, it is not critical and it is trivial... so suggest `app/obolapi` rather